src/glang/driver.py

```diff
@@ -214,6 +240,7 @@ def assemble(self, args: DriverArguments) -> Assembly:
                 f"--mtriple={get_target_triple()}",
                 "--filetype=asm",
                 f"-O{args.optimize}",
+                *(args.extra_llc_args.split(",") if args.extra_llc_args else []),
```

Comment:
Up to you, but you could set a custom `type` (https://docs.python.org/3/library/argparse.html#type) for these two arguments that splits them for you (and then set the default to an empty list)